src/main/resources/db/migration/V0001__create_user.sql

```diff
@@ -43,4 +43,28 @@ CREATE TABLE request(
     description varchar(4000),
     due_date timestamp(6),
     price float(53)
-)
\ No newline at end of file
+);
+
+--contract
+
+CREATE TABLE contract(
+    id BIGSERIAL PRIMARY KEY,
+    status varchar(10) NOT NULL,
```

Comment:
why are there 3 `status`, you only need 2, the last one is deduced from the 2 other